samples/bluetooth/mesh/sensor_client/README.rst

```diff
@@ -15,14 +15,19 @@ In addition, the samples demonstrate usage of both :ref:`single-channel sensor t
    This sample must be paired with :ref:`bluetooth_mesh_sensor_server` to show any functionality.
    The observer has no sensor data, and is dependent on a mesh sensor to provide it.
 
+FEM support
+***********
+
+.. include:: /includes/sample_fem_support.txt
+
 Requirements
 ************
 
 The sample supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :sample-yaml-rows:
+   :sample-yaml-rows: nrf52dk_nrf52832, nrf52840dk_nrf52840, nrf21540dk_nrf52840
```

Comment:
```suggestion
   :sample-yaml-rows:
```

`:sample-yaml-rows:` will automatically extract the list of platforms from sample.yaml, and does not take any arguments.

Same throughout

Reply:
Should we replace all of them with this argument, so we do not have to maintain them. The only thing is that we add 833 and 820 some places.

Reply:
Exactly :)

Reply:
I think so, but IIRC, there was some resistance against this in the docs group, for some reason. Up to @mia-ko

Reply:
But we can not build for the 833 and 820 without changing the sample.yaml file. So it would be good if @mia-ko can find out what we should do.